### DISCUSSION TITLE
How to set defaultValue with styles?

### DISCUSSION THREAD

**Discussion by aldwnesx** [csandman/chakra-react-select#102] [Q&A]
Default values are not given the appropriate color

![image](https://user-images.githubusercontent.com/52225831/148509447-67e3a868-0d3a-4d37-aa45-984fbee606f8.png)

```jsx
 <FormControl p={4}>
  <FormLabel>Select Tag</FormLabel>
  <Select
    size="lg"
    isMulti
    name="colors"
    options={mappedOptions}
    placeholder="Select some tags"
    closeMenuOnSelect={false}
    onChange={handleSetSelectedOptions}
    defaultValue={[mappedOptions[1]]}

  />
</FormControl>
```

**Comment by csandman** [comment]
Can you show me what the data looks like? Maybe a [codesandbox reproduction](https://codesandbox.io/s/chakra-react-select-vsvr0?file=/example.js)

**Comment by csandman** [comment]
I tried making an example as close to the code as you provided as I could, and it appears to work fine:

https://codesandbox.io/s/chakra-react-select-demo-default-option-color-hrgm8?file=/example.js

In order to set a color on a tag, either include the `colorScheme` value in your option object:

```js
const options = [{
  label: "Blue",
  value: "blue",
  colorScheme: "blue",
}]
```

or pass the `colorScheme` prop to the `Select` if you want all of the tags to be styled like that:

```js
return (
  <Select colorScheme="purple" />
);
```
